doc/arkode/guide/source/Usage/SplittingStep/SplittingStepCoefficients.rst

```diff
@@ -0,0 +1,470 @@
+   +--------------------------------------------------------------+-------------------------------------------------------------+
+   | :c:func:`SplittingStepCoefficients_ThirdOrderSuzuki()`       | Create a third order composition method of Suzuki           |
+   +--------------------------------------------------------------+-------------------------------------------------------------+
+   | :c:func:`SplittingStepCoefficients_TripleJump()`             | Create an arbitrary order, three-jump composition method    |
+   +--------------------------------------------------------------+-------------------------------------------------------------+
+   | :c:func:`SplittingStepCoefficients_SuzukiFractal()`          | Create an arbitrary order, five-jump composition method     |
+   +--------------------------------------------------------------+-------------------------------------------------------------+
+   | :c:func:`SplittingStepCoefficients_Alloc()`                  | Allocate an empty SplittingStepCoefficient                  |
+   +--------------------------------------------------------------+-------------------------------------------------------------+
+   | :c:func:`SplittingStepCoefficients_Create()`                 | Create a new SplittingStepCoefficient from coefficients     |
+   +--------------------------------------------------------------+-------------------------------------------------------------+
+   | :c:func:`SplittingStepCoefficients_Copy()`                   | Create a copy of a SplittingStepCoefficients                |
+   +--------------------------------------------------------------+-------------------------------------------------------------+
+   | :c:func:`SplittingStepCoefficients_Free()`                   | Deallocate a SplittingStepCoefficients                      |
+   +--------------------------------------------------------------+-------------------------------------------------------------+
+   | :c:func:`SplittingStepCoefficients_Write()`                  | Write the SplittingStepCoefficients to an output file       |
+   +--------------------------------------------------------------+-------------------------------------------------------------+
+
+
+.. c:function:: SplittingStepCoefficients SplittingStepCoefficients_LoadCoefficients(ARKODE_SplittingCoefficientsID method)
+
+   Retrieves specified splitting coefficients. For further information on the
+   current set of splitting coefficients and their corresponding identifiers,
+   see
+   :numref:`ARKODE.Usage.SplittingStep.SplittingStepCoefficients.Coefficients`.
+
+
+   **Arguments:**
+      * ``method`` -- the splitting coefficients identifier.
+
+   **Return value:**
+      * A :c:type:`SplittingStepCoefficients` structure if successful.
+      * A ``NULL`` pointer if *method* was invalid or an allocation error occurred.
+   
+   .. versionadded:: x.y.z
```

Comment:
Is there a reason this does not use the updated documentation style (e.g., `:param method:`?